uiowa/uiowa
# Remove article caption from render, update help text

Resolves https://github.com/uiowa/uiowa/issues/3461 and fixes regression related to the hero to banner refactor of article featured image.

# How to test

- `yarn workspace uids_base gulp --development`
- Go to https://stanleymuseum.local..drupal.uiowa.edu/news and verify that page overflow scrollbar is gone.
